build.sh

```diff
@@ -18,11 +18,13 @@ ARGS=$*
 # script, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean libcuspatial cuspatial tests benchmarks -v -g -n -h --allgpuarch --show_depr_warn"
+VALIDARGS="clean libcuspatial cuspatial cuproj tests benchmarks -v -g -n -h --allgpuarch --show_depr_warn"
```

Comment:
```suggestion
VALIDARGS="clean libcuspatial cuspatial libcuproj cuproj tests benchmarks -v -g -n -h --allgpuarch --show_depr_warn"
```

Reply:
I made this mistake before. Currently cuProj is built by cuSpatial CMake so this is not needed.